dts/arm/nxp/nxp_lpc55S3x_common.dtsi

```diff
@@ -79,7 +79,7 @@
 		reg = <0x34000 0x1000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		status = "disabled";
+		status = "ok";
```

Comment:
Can we keep this as `status="disabled"`, and enable the IAP at the board level? Doing it this way will give a clear indication that on this board the IAP has been enabled and tested.

Reply:
Well, this FMC + flash is only on the SoC and does not require any pin setting to operate. The declarations belong to the SoC's dtsi. Now, let's not confuse device tree for documentation. There is the board's doc folder for that.
Your comment highlights a failure of this PR though, the lack of documentation update... That needs to be addressed.